packages/react-native-client/src/common/client.ts

```diff
@@ -1,12 +1,21 @@
 import { Metadata } from '../types';
 import RNFishjamClientModule from '../RNFishjamClientModule';
 
+export type Config = {
+  reconnectConfig?: {
+    maxAttempts?: number;
+    initialDelayMs?: number;
+    delayMs?: number;
```

Comment:
Just an idea, I'm not sure about it:
can we make these values mandatory in native code, but define default values in RN SDK? This way there will be single place with default values (right now there are 3+?), but they will not be mandatory for RN SDK users (but they will be mandatory for native SDKs users...)

Reply:
Record fields in expo modules must have a default value: https://docs.expo.dev/modules/module-api/#records
And making the values mandatory for native SDK users would be mean for them. I'd leave it as it is, if we'd like to change this we'd use search option anyway.